
<launch>
  <arg name="object_name" default="auk" />
    <!-- Ros Vrpn Estimation node -->
  <node ns="$(arg object_name)" name="vicon_estimator" type="vicon_estimation_node" pkg="ros_vrpn_client" output="screen">
      <param name="object_name" value="$(arg object_name)" />
      <param name="verbose" value="true" />
      <param name="translational_estimator/kp" value="0.15" />
      <param name="translational_estimator/kv" value="0.5" />
      <param name="translational_estimator/outlier_threshold_meters" value="0.25" />
      <param name="translational_estimator/maximum_outlier_count" value="30" />
      <param name="rotational_estimator/orientation_estimate_initial_covariance" value="10.0" />
      <param name="rotational_estimator/rate_estimate_initial_covariance" value="10.0" />
      <param name="rotational_estimator/orientation_process_covariance" value="0.000002" />
      <param name="rotational_estimator/rate_process_covariance" value="10.0" />
      <param name="rotational_estimator/orientation_measurementCovariance" value="0.001" />
      <param name="rotational_estimator/outlier_rejection_method" value="mahalanobis_distance" />
      <param name="rotational_estimator/outlier_rejection_mahalanobis_threshold" value="4.0" />
      <param name="rotational_estimator/outlier_rejection_subsequent_threshold_degrees" value="30.0" />
      <param name="rotational_estimator/outlier_rejection_subsequent_maximum_count" value="50.0" />
      <param name="rotational_estimator/output_minimal_quaternions" value="false" />
    </node>
</launch>
